### PR TITLE
Add post papers

### PR DESCRIPTION
**Pull request by beatrizserrano**

### CODE REVIEW

File: _posts/2021-05-13-paper-cardio.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+site: freiburg
+title: 'New Paper "Novel insights into the electrophysiology of murine cardiac macrophages: relevance of voltage-gated potassium channels"'
+doi: "10.1093/cvr/cvab126"
+tags:
+- paper
+---
+
+
+_Abstract_ - Macrophages (MΦ), known for immunological roles such as phagocytosis and antigen presentation, have been found to electrotonically couple to cardiomyocytes (CM) of the atrio-ventricular node via Cx43, affecting cardiac conduction in isolated mouse hearts. Here, we characterise passive and active electrophysiological properties of murine cardiac resident MΦ, and model their potential electrophysiological relevance for CM.
```
Comment (bgruening) [inline _posts/2021-05-13-paper-cardio.md:10]: we usually name the author and give the group shout out

File: _posts/2021-05-13-paper-cardio.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+site: freiburg
+title: 'New Paper "Novel insights into the electrophysiology of murine cardiac macrophages: relevance of voltage-gated potassium channels"'
+doi: "10.1093/cvr/cvab126"
+tags:
+- paper
```
Comment (bgruening) [inline _posts/2021-05-13-paper-cardio.md:6]: the sfb1452 as the one that created this paper

Comment (beatrizserrano) [reply _posts/2021-05-13-paper-cardio.md:6]: supporter then

File: _posts/2021-05-14-papers-teaching.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+<br>
+The [second manuscript](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008923#sec003) describes how Galaxy can facilitate online teaching using the Galaxy Training Network materials, Training Infrastructure as a Service and many features such as data, history and workflow sharing, with a concrete proposed format to make the most of your training events.
+
+<br>
+
+{% twitter https://twitter.com/gxytraining/status/1393104039236317188 %}
+
+<br>
+
+Read them both to get some practical tips for your next training!
```
Comment (bgruening) [inline _posts/2021-05-14-papers-teaching.md:27]: Do you want to mentione that we already used those recommendations in 3 highly sucessful workshops?

File: _posts/2021-05-13-paper-cardio.md (hunk 2 above)
Comment (beatrizserrano) [reply _posts/2021-05-13-paper-cardio.md:6]: mentioned in the text

File: _posts/2021-05-13-paper-cardio.md (hunk 1 above)
Comment (beatrizserrano) [reply _posts/2021-05-13-paper-cardio.md:10]: done

File: _posts/2021-05-14-papers-teaching.md (hunk 3 above)
Comment (beatrizserrano) [reply _posts/2021-05-14-papers-teaching.md]: done too

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Danke!

**Pull request merged**